.github/workflows/docker.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           echo "CLIENT_PORT=${{ vars.CLIENT_PORT }}" >> $GITHUB_ENV
           echo "SERVER_HOST=${{ vars.SERVER_HOST }}" >> $GITHUB_ENV
           echo "SERVER_PORT=${{ vars.SERVER_PORT }}" >> $GITHUB_ENV
+          echo "SOCKET_GW_PORT=${{ vars.SOCKET_GW_PORT }}" >> $GITHUB_ENV
```

Comment:
I see there is a lack of some variables in the workflow. Should we consider adding these?

```yaml
echo "APP_ENV=${{ vars.APP_ENV }}" >> $GITHUB_ENV
echo "NEXT_GW_SERVER_HOST=${{ vars.NEXT_GW_SERVER_HOST }}" >> $GITHUB_ENV
echo "CORS_ORIGIN=${{ vars.CORS_ORIGIN }}" >> $GITHUB_ENV
echo "GOOGLE_EMAIL=${{ vars.GOOGLE_EMAIL }}" >> $GITHUB_ENV
echo "GOOGLE_PASSWORD=${{ vars.GOOGLE_PASSWORD }}" >> $GITHUB_ENV
```

Reply:
Doesn't make any difference, but change.